config/webpack/test.js

```diff
@@ -1,5 +1,6 @@
 process.env.NODE_ENV = process.env.NODE_ENV || 'development'
 
 const environment = require('./environment')
-
+var babelLoader = environment.loaders.get('babel')
+babelLoader.include.push(babelLoader.include[0].concat('../../test'))
```

Comment:
If we want to change the location of the tests, editing this line in addition to the relevant `scripts` line in the package.json will be enough to do it